packages/xo-server/src/xapi/index.js

```diff
@@ -1406,7 +1407,7 @@ export default class Xapi extends XapiBase {
     vmId,
     hostXapi,
     hostId,
-    { sr, migrationNetworkId, mapVifsNetworks, mapVdisSrs } = {}
+    { force = false, mapVdisSrs, mapVifsNetworks, migrationNetworkId, sr } = {}
```

Comment:
Please check at which level default values are usually set and do the same. I don't think it should be defined 3 times.